src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -0,0 +1,214 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import logging
+import math
+import os
+import sas
+import sas_testcase
+from reference_models.propagation import wf_itm
+from reference_models.propagation import wf_hybrid
+from reference_models.antenna import antenna
+from reference_models.ppa import ppa
+nlcd_driver = ppa.nlcd_driver
+from util import winnforum_testcase, configurable_testcase, writeConfig, loadConfig
+from reference_models.geo import utils as GeoUtils
+
+def computePropagationAntennaModel(request):
+    reliability_level = request['reliabilityLevel']
+    if reliability_level not in [-1, 0.05, 0.95]:
+        raise ValueError('reliability_level not in [-1, 0.05, 0.95]')
+
+    tx = request['cbsd']
+    if ('fss' in request) and ('ppa' in request):
+        raise ValueError('fss and ppa in request')
+    elif 'ppa' in request:
+        rx ={}
+        rx['height'] = 1.5
+        isfss = False
+        coordinates = []
+        ppa = request['ppa']
+
+        ARCSEC = 2
```

Comment:
One thing that came up during the WG4 call today is the following:
 - PPA protection points are every 2 arcseconds
 - NLCD data is sampled every 1 arcsecond
 - By their definitions, these grids will coincide (i.e. every PPA point will have a corresponding NLCD sample point at exactly the same location)

In order to keep things simple, we agreed on the call this morning to limit the input polygon to contain only one ONE-arcsecond grid point. With that modification, the code that you have works perfectly (otherwise you would have needed to give different inputs to `RegionNlcdVote`).

So the only change now is to set `arcsec = 1` so that configuration errors are caught.